jwks-slim/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "jwks-slim",
+  "version": "0.0.1-jwks-client.8",
+  "description": "Library to retrieve RSA public keys from a JWKS endpoint",
+  "main": "src/index.js",
+  "scripts": {
+    "test:node": "jest --verbose --detectOpenHandles --env=node",
+    "test:worker": "jest --verbose --detectOpenHandles --env=@edge-runtime/jest-environment",
+    "test": "npm run test:node && npm run test:worker"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@edge-runtime/jest-environment": "^2.3.10",
```

Comment:
As it's Jest related, could this be in devDependencies instead

Reply:
done